index.js

```diff
@@ -0,0 +1,243 @@
+  hamburger.classList.remove('hide');
+});
+
+// popup
+const portfolio = document.querySelector('#portfolio');
+const modal = document.querySelector('#myModal');
+
+portfolio.classList.add('works');
+
+Array.from(portfolio.children).forEach((child, index) => {
+  child.innerHTML = `
+  <section class="${data[index].card}">
+    <ul>
+      <li><img src="${data[index].image.mobile}" alt="icon" class="img-1-1" /></li>
+      <li><img src="${data[index].image.desktop}" alt="icon" class="img-1-2" /></li>
+    </ul>
+    <div class="block-1">
+      <div class="text-group">
+        <h2 class="lg-text-2">${data[index].name}</h2>
+        <div class="client-info">
+          <h3 class="client">${data[index].job.client}</h3>
+          <a href="#"><img src="./assets/Counter.svg" alt="counter" /></a>
+          <h3 class="role">${data[index].job.role}</h3>
+          <a href="#"><img src="./assets/Counter.svg" alt="counter" /></a>
+          <h3 class="year-label">${data[index].job.year}</h3>
+        </div>
+      </div>
+      <h3 class="sm-text-2">${data[index].description.summary}</h3>
+      <ul class="tags">
+        <li>${data[index].technologies.html}</li>
+        <li>${data[index].technologies.css}</li>
+        <li>${data[index].technologies.javascript}</li>
+      </ul>
+      <div class="actions" id="${data[index].id}">
+        <button class="btn" id="btn" type="button">See Project</button>
+      </div>
+    </div>
+  </section>`;
+});
+
+// modals
+Array.from(portfolio.children).forEach((item, index) => {
+  item.firstElementChild.lastElementChild.lastElementChild.firstElementChild.addEventListener('click', () => {
+    modal.innerHTML = `
+    // <!-- Modal content -->
+    <section class="modal-content">
+      <div class="popup-text-group">
+        <div class="text-group">
+          <h2 class="popup-text-lg">${data[index].name}</h2>
+          <div class="client-info">
+            <h3 class="client">${data[index].job.client}</h3>
+            <a href="#"><img src="./assets/Counter.svg" alt="counter" /></a>
+            <h3 class="role">${data[index].job.role}</h3>
+            <a href="#"><img src="./assets/Counter.svg" alt="counter" /></a>
+            <h3 class="year-label">${data[index].job.year}</h3>
+          </div>
+        </div>
+        <span class="close">&times;</span>
+      </div>
+      <ul>
+        <li><img src="${data[index].image.mobile}" alt="icon" class="img-1-1" /></li>
+        <li><img src="${data[index].image.desktop}" alt="icon" class="img-1-2 popup-img" /></li>
+      </ul>
+      <div class="popup-bottom">
+        <p class="popup-text-sm">${data[index].description.portfolio}</p>
+        <div class="popup-block">
+          <ul class="tags ${data[index].id}">
+          </ul>
+          <div class="popup-actions">
+            <button class="btn btnImg" id="btn" type="button">See Live
+              <img src="./assets/Icon.svg" alt="icon" id="imgIcon" />
+            </button>
```

Comment:
- [ ] In the popup window, the `see live` is not showing. kindly fix it. :+1: 

![icon](https://user-images.githubusercontent.com/70483835/188826192-98c6456d-7f1e-4302-88f6-17a56cf1f873.png)